databricks/sdk/mixins/jobs.py

```diff
@@ -0,0 +1,31 @@
+from databricks.sdk.service import jobs
+
+
+class JobsExt(jobs.JobsAPI):
+
+    def get_run(self,
+                run_id: int,
+                *,
+                include_history: bool | None = None,
+                include_resolved_values: bool | None = None,
+                page_token: str | None = None) -> jobs.Run:
```

Comment:
I believe the syntax you're going for is something in the lines of `Optional[type]`